src/components/ModalExport.tsx

```diff
@@ -210,9 +209,38 @@ const ModalExport = () => {
     setPublishing(false)
   }
 
+  const [advancedSettings, setAdvancedSettings] = useState(false)
+
+  /** Toggles advanced setting when Advanced CTA is clicked. */
+  const onAdvancedClick = () => setAdvancedSettings(!advancedSettings)
+
+  const [shouldIncludeMetaAttributes, setShouldIncludeMetaAttributes] = useState(true)
+
+  /** Updates checkbox value when clicked and set the appropriate value in the selected option. */
+  const onChangeLosslessCheckbox = () => {
+    const updatedValue = {
+      ...selected,
+      excludeMeta: !shouldIncludeMetaAttributes
+    }
+    setSelected(updatedValue)
+    setShouldIncludeMetaAttributes(!shouldIncludeMetaAttributes)
+  }
+
+  /** Created an array of objects so that we can just add object here to get multiple checkbox options created. */
+  const checkboxArray = [
+    {
+      id: 'lossless-checkbox',
+      onChangeFunc: onChangeLosslessCheckbox,
+      seq: 1,
```

Comment:
Let's call this `defaultChecked`. I'm not sure what `seq` means.

Reply:
Done